packages/frontend/src/app.tsx

```diff
@@ -9,6 +10,7 @@ export function App() {
       setAPI={console.log}
     >
       <p>I am the child</p>
+      <SelectSport />
```

Comment:
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH


Please remove this file shouldn't be changed